contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3926,6 +3926,101 @@ RETURNS SETOF RECORD
 AS 'babelfishpg_tsql' LANGUAGE C STABLE;
 GRANT EXECUTE ON FUNCTION sys.fn_listextendedproperty TO PUBLIC;
 
+-- Matches and returns column length of the corresponding column of the given table
+CREATE OR REPLACE FUNCTION sys.COL_LENGTH(IN object_name TEXT, IN column_name TEXT)
+RETURNS SMALLINT AS $BODY$
+    DECLARE
+        col_name TEXT;
+        object_id oid;
```

Comment:
From which schema are we using text data type from ?

Reply:
from `pg_catalog` schema